internal/server/ranking/ranking.go

```diff
@@ -139,9 +139,9 @@ func (a CohortByRank) Swap(i, j int) {
 
 func (a CohortByRank) Less(i, j int) bool {
 	oi := a[i]
-	scorei := getScorePb(oi)
+	scorei := GetScorePb(oi)
 	oj := a[j]
-	scorej := getScorePb(oj)
+	scorej := GetScorePb(oj)
 	// Higher score value means lower rank.
```

Comment:
Should we drop cohort "place coverage" ranking to be aligned with BQ results?

Reply:
Done